content/posts/marimo.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "marimo: a reactive Python notebook that runs in the browser"
+date: 2024-03-28T14:57:24-07:00
+draft: true
+tags: ["announcement", "guest post"]
+# author: "Me"
+author: ["Akshay Agrawal", "Myles Scolnick"] # multiple authors
+showToc: true
+TocOpen: false
+draft: false
+hidemeta: false
+comments: false
+description: "Bringing a whole new meaning to \"serverless\"."
+# canonicalURL: "https://canonical.url/to/page"
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+cover:
+    # image: "<image path/url>" # image path/url
+    # alt: "<alt text>" # alt text
+    # caption: "<text>" # display caption under cover
+    relative: false # when using page bundles set this to true
+    hidden: true # only hide on current single page
+---
+
+
+We’re [Akshay Agrawal](https://akshayagrawal.com) and [Myles
+Scolnick](https://mylesscolnick.com/), and we’re the developers of
+[marimo](https://github.com/marimo-team/marimo), an open-source reactive
+notebook for Python. marimo is reproducible (no hidden state), stored as a Python
+file (easily versioned with git), executable as a script, and deployable as
+an interactive web app.
+
+Thanks to Pyodide, anyone in the world can create and share
+marimo notebooks at our [online marimo
+playground](https://marimo.new) for free, without having to install Python
+or pay for infrastructure. The playground uses Pyodide to run marimo
+entirely in the browser: instead of shipping Python to a remote server for
+execution, your browser runs it directly on your computer.
+
+As an example, here's our tutorial notebook running via Pyodide:
+
+<iframe
+  src="https://marimo.app/l/c7h6pz?embed=true"
+  width="800"
+  height="700"
+  style=" max-width: 800px"
+></iframe>
+
+In this post, we'll describe how we ported marimo to work via
+Pyodide, and why we're incredibly excited about the potential of Pyodide
+to make computing more accessible.
+
+## What is marimo?
+
+marimo is an open-source reactive notebook for Python built from the ground up to solve [well-known problems with traditional notebooks](https://docs.marimo.io/faq.html#faq-problems), including reproducibility, interactivity, maintainability, and shareability.
+
+
+**A reactive notebook with no hidden state.**
+In marimo, a notebook’s code, outputs, and program state are always consistent.
+Run a cell and marimo **reacts** by automatically running the cells that
```

Comment:
I think italics is better here since we used bold for the heading:
```suggestion
Run a cell and marimo _reacts_ by automatically running the cells that
```